.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,6 +16,25 @@ Delete any items that are not applicable to this PR.
 - [ ] This renders correctly on smaller devices using a responsive layout. (You can test this [in your browser](https://www.browserstack.com/guide/responsive-testing-on-local-server))
 - [ ] This was checked for [cross-browser compatibility](https://www.elastic.co/support/matrix#matrix_browsers)
 
+
+### Risk Matrix
```

Comment:
This is not intended as a blocker for this PR, but a question about this template generally which might resonate with others:

Is this not growing a bit too long? Mainly in the sense that the PR description, before adding a description of the work takes up a lot of screen space and presents all information as equally important (see [after current changes](https://github.com/streamich/kibana/blob/risk-matrix/.github/PULL_REQUEST_TEMPLATE.md))

I think something like this would be cool (using "details" for the stuff we expect to see on every PR):

![Screenshot 2021-05-27 at 12 42 30](https://user-images.githubusercontent.com/8155004/119813009-13932580-bee9-11eb-899a-a25caebad3b0.png)

Reply:
To limit the size impact on the PR template, we have a separate list of risks in `RISK_MATRIX.mdx`.

I like the idea of foldable sections. But to limit the scope of this PR to just the "Risk Matrix" section, I will pass on this change for this PR, but in general, :+1: for adding some way of foldable sections to the template.